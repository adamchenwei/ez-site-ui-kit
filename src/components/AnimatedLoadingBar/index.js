import React from 'react';
import Bar from './style/Bar';

const AnimatedLoadingBar = () => <Bar />;

export default AnimatedLoadingBar;
