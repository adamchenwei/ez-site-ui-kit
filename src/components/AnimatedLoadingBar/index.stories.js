import React from 'react';
import { storiesOf } from '@storybook/react';
import AnimatedLoadingBar from './index';

storiesOf('Component/AnimatedLoadingBar/Stand Alone', module).add('default', () => (
  <AnimatedLoadingBar />
));
