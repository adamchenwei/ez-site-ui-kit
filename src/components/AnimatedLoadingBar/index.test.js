import React from 'react';
import renderer from 'react-test-renderer';
import AnimatedLoadingBar from './index';

describe('AnimatedLoadingBar', () => {
  test('renders as expected', () => {
    const tree = renderer.create(<AnimatedLoadingBar />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
