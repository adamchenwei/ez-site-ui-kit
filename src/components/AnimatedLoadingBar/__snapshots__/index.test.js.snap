// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AnimatedLoadingBar renders as expected 1`] = `
<div
  className="Bar-gSSpUe eThZd"
/>
`;
